### PR TITLE
Added sepia filtered image

### LINKED ISSUE #335: Converting image into sepia filtered image
# Description

**HackinCodes2020 Participant**

The sepia filter is one of the most popular tools for editing images.
Widely used by photographers back in the early days of photography
I want to implement a script that will read the original image and convert the original image into a sepia filtered image.
## Language

- [x] Python

## Type of issue

- [x] Feature

## Checklist:

- **For Feature**

| name            | about                                                  | labels  | assignees                                                      |
| --------------- | ------------------------------------------------------ | ------- | -------------------------------------------------------------- |
| Sepia Filtered Image | Make the image look old antique picture | feature | @sharur7  |

**Additional context**:  Sepia effect will help the image to add archival qualities.
@srimani-programmer @HarshCasper @seema1711  I want to work on this issue.

### PR DESCRIPTION
**Pull request by sharur7**
# Description

The original image is converted into a Sepia filtered image.

Fixes #335 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by sharur7** [comment]
@HarshCasper @srimani-programmer please review my PR

File: Python/Image To Sepia Effect/sepia image convertor.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Importing libraries
+import cv2
+import numpy as np
+
+img = cv2.imread('taj_mahal.jpg')
```
Comment (HarshCasper) [inline Python/Image To Sepia Effect/sepia image convertor.py:5]: Take the input from the user dynamically

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Kindly make the mentioned changes

File: Python/Image To Sepia Effect/sepia image convertor.py (hunk 1 above)
Comment (sharur7) [reply Python/Image To Sepia Effect/sepia image convertor.py:5]: How should I do that?

Comment (sharur7) [reply Python/Image To Sepia Effect/sepia image convertor.py:5]: I tried using Pillow but that's not working :(

Comment (HarshCasper) [reply Python/Image To Sepia Effect/sepia image convertor.py:5]: Well, don't you think, it would be redundant if the user cannot pass it dynamically?

Comment (sharur7) [reply Python/Image To Sepia Effect/sepia image convertor.py]: Yep, now it worked! Can you please review it again?

File: Python/Image To Sepia Effect/sepia image convertor.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Importing libraries
+import cv2
+import numpy as np
+from PIL import Image
+
+# Taking input from the user
+filename = input("Enter the file name: ")
+img = Image.open("C:/Users/Sharvari/Desktop/Internships/Courses & Events/HackinCodes/Image To Sepia Effect/Image To Sepia Effect/"+ filename)
```
Comment (HarshCasper) [inline Python/Image To Sepia Effect/sepia image convertor.py:8]: Remove this. It's simply redundant as it references your Local Machine Configurations

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: Python/Image To Sepia Effect/sepia image convertor.py (hunk 2 above)
Comment (sharur7) [reply Python/Image To Sepia Effect/sepia image convertor.py]: Done

**Pull request merged**